### PR TITLE
Added initCubeTexture to WebGLRenderer.

### PR DESCRIPTION
**Pull request by amsXYZ**
Just as ``WebGLRenderer.initTexture`` is useful to upload loaded maps to the GPU without having to wait for first use, I propose ``WebGLRenderer.initCubeTexture`` to upload cubemaps to the GPU without the need to use them first (i.e. assigning them to ``Scene.background``).

### CODE REVIEW

**Comment by Mugen87** [comment]
Related #17697.

I'm okay with this enhancement as long as the docs and the TS file are also updated.

**Comment by amsXYZ** [comment]
Sure, I will shortly update the TS and docs. Didn't do it the first time since it was missing for ``initTexture`` and thought that might still be experimental API.

**Comment by amsXYZ** [comment]
Added types and docs, please re-review @Mugen87

File: src/renderers/WebGLRenderer.d.ts [hunk 1]
```diff
@@ -430,6 +430,13 @@ export class WebGLRenderer implements Renderer {
 	 */
 	initTexture( texture: Texture ): void;
 
+	/**
+	 * Initializes the given cube texture. Can be used to preload a cubemap rather than waiting until first render (which can cause noticeable lags due to decode and GPU upload overhead).
+	 *
+	 * @param texture The cube texture to Initialize.
+	 */
+	initCubeTexture( texture: CubeTexture ): void;
```
Comment (Mugen87) [inline src/renderers/WebGLRenderer.d.ts]: You have to import `CubeTexture` before you can use it.

Comment (amsXYZ) [reply src/renderers/WebGLRenderer.d.ts]: Sorry for missing this! Already resolved.

**Review by Mugen87 — APPROVED** [review]

**Comment by Mugen87** [comment]
Closing for now. We can reopen the PR if more developers request this method 👍 .

**Pull request closed without merging**